pagopa/io-app
# fix: [#176740861] Clean entities/services/readState reducer when services change

## Short description
The reducer is now cleaned upon receiving new services.

## How to test
- Open the app, go to tab "services" and then "all"
- State should look like:
<img width="756" alt="before" src="https://user-images.githubusercontent.com/26572302/108896384-23af1680-7615-11eb-8599-d767cf9ece8b.png">


- Change `mock-server/src/payloads/service.ts` line `48`
- Restart the mock server
- On the services tab, pull to refresh
- See the new services and the existing ones mantaining their state:

<img width="755" alt="after" src="https://user-images.githubusercontent.com/26572302/108896492-48a38980-7615-11eb-8be1-14505f711bbb.png">
